### DISCUSSION TITLE
AAD SSO z SAML - brak URL do inicjowania połączenia

### DISCUSSION THREAD

**Discussion by ithmpl** [AzureCommunityPL/group#238] [Pomoc] [answered]
Drodzy,

Konfiguruje SSO do aplikacji zewnętrznego klienta z użyciem SAML.
Sama integracja działa prawidłowo z użyciem opcje testowania połaczenia w Enterprise Application. 

Ale dostawca aplikacji nie ma wersji, gdzie inicjacja połączenia odbywa się z URL tej aplikacji. Pierwszy raz się z tym spotkałem. 

Dostałem  taką podpowiedź od dostawcy:
„Muszą Państwo podać użytkownikom swój wewnętrzny URL skonfigurowanego połączenia FederationServices.”

Podpowiecie proszę jaki to będzie URL z AAD? 

Ewentualnie czy istniej możliwość utworzenie aplikacji, które jedynie co będzie robiła to właśnie to przekierowanie?

**Comment by pawelserwan** [comment, marked as answer]
hej,

czyli wychodzi na to że wspierają tylko scenariusz IdP Initiaited Request. W związku z tym najlepiej jak podasz im User access URL z Properties Twojej Enterprise Application - zaczyna się od https://myapps.*
Daj znac czy działa :)

**Reply by ithmpl** [reply]
Dziękuje, tego potrzebowałem.

**Reply by jpodoba** [reply]
Hey nie wiem czy dobrze zrozumiałem ale prawdopodobnie dostwca aplikacji pytał gdzie mają kierować SAML requesty. Na początku musisz sobie utworzyć Enterprise app. Idąc dalej masz dwie opcje zależne od tego czy ta aplikacja przewiduje współpracę z wieloma tenantami.

1. single tenant, w konfigracji **Single sign-on** wyciągasz sobie **Login URL** (https://login.microsoftonline.com/1111-1111-1111-1111-1111111111/saml2)

2. multi tenant, wtedy podajesz adres multiplexera https://login.microsoftonline.com/common w tym przypadku Azure AD sam rozpozna do którego tenanta należy użytkownik i przekieruje request w dobre miejsce.

Więcej masz tutaj: https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant
